tests/baselines/reference/arrayLiteralTypeInference.errors.txt

```diff
@@ -0,0 +1,72 @@
+tests/cases/compiler/arrayLiteralTypeInference.ts(13,5): error TS2322: Type '({ id: number; trueness: boolean; } | { id: number; name: string; })[]' is not assignable to type 'Action[]'.
+  Type '{ id: number; trueness: boolean; } | { id: number; name: string; }' is not assignable to type 'Action'.
+    Type '{ id: number; trueness: boolean; }' is not assignable to type 'Action'.
+      Property 'trueness' does not exist on type 'Action'.
+tests/cases/compiler/arrayLiteralTypeInference.ts(29,5): error TS2322: Type '({ id: number; trueness: boolean; } | { id: number; name: string; })[]' is not assignable to type '{ id: number; }[]'.
+  Type '{ id: number; trueness: boolean; } | { id: number; name: string; }' is not assignable to type '{ id: number; }'.
+    Type '{ id: number; trueness: boolean; }' is not assignable to type '{ id: number; }'.
+      Property 'trueness' does not exist on type '{ id: number; }'.
+
+
+==== tests/cases/compiler/arrayLiteralTypeInference.ts (2 errors) ====
+    class Action {
+        id: number;
+    }
+    
+    class ActionA extends Action {
+        value: string;
+    }
+    
+    class ActionB extends Action {
+        trueNess: boolean;

```

Comment:
I think this should be `trueness` -- it doesn't look like this test was intended to fail. Another :trophy: for this change's effectiveness.

Reply:
https://stackoverflow.com/questions/50143250/how-excess-property-check-helps